src/Surface.zig

```diff
@@ -274,6 +274,7 @@ const DerivedConfig = struct {
     mouse_shift_capture: configpkg.MouseShiftCapture,
     macos_non_native_fullscreen: configpkg.NonNativeFullscreen,
     macos_option_as_alt: ?input.OptionAsAlt,
+    macos_option_as_alt_original: ?input.OptionAsAlt,
```

Comment:
Happy to learn if there is a better way to store the "original" state before the user triggers the binding action. 😄